backend/algorithms/errors.json

```diff
@@ -7211,127 +7578,121 @@
-            "handbook_note": "You must be enrolled in a CSE BE (Hons) program and complete all third year core requirements to enrol in this course."
+            "original": "Pre-requisite: COMM1140. Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Terms 1 and 3.<br/><br/>",
+            "processed": "COMM1140. Only available to single && double degree Business School in Term 2. It will be offered to non-Business School in Terms 1 && 3"
         },
         "tokens": [
             "(",
-            "126UOC",
+            "COMM1140.",
+            "Only",
+            "available",
+            "to",
+            "single",
             "&&",
-            "(",
-            "BINF?H",
-            "||",
-            "SENGAH",
-            "||",
-            "COMP?H",
-            ")",
+            "double",
+            "degree",
+            "Business",
+            "School",
+            "in",
+            "Term",
+            "2.",
+            "It",
+            "will",
+            "be",
+            "offered",
+            "to",
+            "non-Business",
+            "School",
+            "in",
+            "Terms",
+            "1",
+            "&&",
+            "3",
             ")"
         ],
-        "broke at": "Index 1, 126UOC"
+        "broke at": "Index 1, COMM1140."
     },
-    "COMP9302": {
+    "COMM3091": {
         "condition": {
-            "original": "(COMP6441 OR COMP6841) AND (12 UOC from (COMP6443, COMP6843, COMP6445, COMP6845, COMP6447)) AND enrolled in final term of program<br/><br/>",
-            "processed": "(COMP6441 || COMP6841) && (12UOC in (COMP6443 || COMP6843 || COMP6445 || COMP6845 || COMP6447))",
-            "handbook_note": "This course can only be taken in the final term of your program."
+            "original": "Completed at least 72 UoC and be enrolled in a Commerce Program; be in good academic standing, and completed COMM1999<br/><br/>",
+            "processed": "72UOC && be a Commerce Program; be in good academic standing && COMM1999"
         },
         "tokens": [
             "(",
-            "(",
-            "COMP6441",
-            "||",
-            "COMP6841",
-            ")",
+            "72UOC",
             "&&",
-            "(",
-            "12UOC",
+            "be",
+            "a",
+            "Commerce",
+            "Program;",
+            "be",
             "in",
-            "(",
-            "COMP6443",
-            "||",
-            "COMP6843",
-            "||",
-            "COMP6445",
-            "||",
-            "COMP6845",
-            "||",
-            "COMP6447",
-            ")",
-            ")",
+            "good",
+            "academic",
+            "standing",
+            "&&",
+            "COMM1999",
             ")"
         ],
-        "broke at": "Index 1, ("
+        "broke at": "Index 3, be"
     },
-    "COMP9491": {
+    "COMP4951": {
```

Comment:
oh no the UOC bug strikes again... Ill make sure to look at this after you are done with this PR.

Reply:
hmm i feel like this should be fixed as part of my PR, e.g. AERO4110 broke as well

Reply:
sure :)

Reply:
nvm this isn't a UOC issue, it's an honours program issue.

Reply:
Ah cool